app/src/main/java/org/oppia/app/help/FAQ/FAQActivity.kt

```diff
@@ -0,0 +1,29 @@
+package org.oppia.app.help.FAQ
+
+import android.content.Context
+import android.content.Intent
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import org.oppia.app.R
+import org.oppia.app.activity.InjectableAppCompatActivity
+import javax.inject.Inject
+
+class FAQActivity : InjectableAppCompatActivity() {
+
+  @Inject
+  lateinit var FAQActivityPresenter: FAQActivityPresenter
```

Comment:
This should come in single line:
`@Inject lateinit var FAQActivityPresenter: FAQActivityPresenter`